scripts/run-burnin.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+
+:START
+
+python scripts\Enlighten.py --log-level debug --plugin Prod.BurnIn 1>>enlighten.out 2>>enlighten.err
+
+echo Sleeping 10 sec...ctrl-C to cancel
+sleep 10
+
+goto START
```

Comment:
Having opened with an eval, let's close with a goto. I have no shame :-(

Reply:
I recently wrote a C program with goto in it. 
(Not for work)

It was equal parts liberating and buggy.